src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,27 @@
+package commands;
+
+import tasks.Task;
+import tasks.TaskList;
+import utils.Storage;
+import utils.Ui;
+
+public class AddCommand extends Command{
+
+    private static final String NEW_TASK_MSG = "New task added:";
+
+    private Task task;
+    
+    public AddCommand(Task task) {
+        this.task = task;
+    }
+
+    public boolean execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Could include the annotation to alert reader that it is overriding the abstract method
```suggestion
    @Override
    public boolean execute(TaskList tasks, Ui ui, Storage storage) {
```

Reply:
Nice catch! thanks